pack/winter.json

```diff
@@ -399,7 +399,7 @@
         "position": 22,
         "quantity": 2,
         "resource_physical": 1,
-        "text": "Team-up (Captain America and Winter Soldier). Max 1 per deck.\n<b>Hero Action</b> <i>(attack)</i>: Deal 6 damage to an enemy. Give Captain America and Winter Soldier each a tough status card.",
+        "text": "Team-Up (Captain America and Winter Soldier). Max 1 per deck.\n<b>Hero Action</b> <i>(attack)</i>: Deal 6 damage to an enemy. Give Captain America and Winter Soldier each a tough status card.",
```

Comment:
The current text (`Team-up`) on these two team-up cards in this pack match the released card images.

I agree that those should probably be `Team-Up` to be consistent with the rest of the cards but I think we should wait for a new errata version to address that instead of making a change now without evidence the designers want it changed.